eos-tech-support/eos-download-image

```diff
@@ -282,63 +284,68 @@ def fetch_windows_tool(session, args, base):
     download(session, url, dest)
 
 
-def fetch_image(session, args, base):
+def fetch_manifest(session, args, base):
     manifest_url = '{base}/releases-{product}-3.json'.format(
         base=base, product=args.product)
     manifest = session.get(manifest_url).json()
 
     images = list(manifest['images'].values())
     images.sort(key=lambda i: distutils.version.LooseVersion(i['version']))
-    if args.mirror:
-        selected = images
-    elif args.version is not None:
+    return manifest, images
+
+
+def fetch_image(session, args, base):
+    _manifest, images = fetch_manifest(session, args, base)
```

Comment:
It does look neater this way, so I've made this change. It's functionally equivalent though:
```python
>>> _, y = (1, 2)
>>> print(_)
1
```